daemon/api.go

```diff
@@ -714,24 +687,50 @@ func modeFlags(devMode, jailMode bool) (snapstate.Flags, error) {
 
 }
 
+func snapUpdateMany(inst *snapInstruction, st *state.State) (msg string, updated []string, tasksets []*state.TaskSet, err error) {
+	// TODO: check inst flags and bail if any are given (-many
+	// doesn't take options)
+	updated, tts, err := snapstateUpdateMany(st, inst.Snaps, inst.userID)
```

Comment:
(in my defence, `tts` is the name of the variable doing exactly the same job on the overlord side --- I know this is not a very good defence though, and am fixing it there as well)